pages/en/lb4/Getting-started.md

```diff
@@ -108,15 +253,11 @@ To run the application, enter:
 npm start
```

Comment:
This is using `npm start` script mentioned by my earlier comment. I think it's not defined if users follow the instructions in the revision of this document proposed by this pull request.